.eslintrc

```diff
@@ -12,6 +15,7 @@
     "import/newline-after-import": "off",
     "import/no-extraneous-dependencies": "off",
     "import/prefer-default-export": "off",
+    "linebreak-style": "off",
```

Comment:
Should we adopt the recommendation outlined [here](https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems) instead of turning the rule off completely?

Reply:
Hm, why though? I'd rather not mess with git's way of doing it - I like it. "check out with Windows line endings, check in with unix" - I don't have to think about it at all. So I don't need the linting - git handles it. Do you need the linting?

Reply:
No I don't think so. I just wanted to ask the question and see if it was considered. I think turning it off will be fine.